src/content/1.4/code/ocaml/snippet04.ml

```diff
@@ -1 +1,4 @@
-let pure x = x, ""
+let ( >=> ) m1 m2 = fun x ->
```

Comment:
The snippets are starting to diverge here but I hope they are in the right place in the book

Reply:
Indeed, the best way would probably be to run the LaTeX compilation and check the output, but from reading along the chapter I believe the sequence of the code samples now makes sense.